### ISSUE TITLE
[docs] Ambiguity in npm-ci

### ISSUE THREAD

**Issue by orjandh** [npm/cli#3036]
The documentation for npm-ci says that the command will be significantly faster if there is a package-lock or npm-shrinkwrap file, but then it says that it must have an existing package-lock or npm-shrinkwrap. If the latter is true, the first point should be removed.

Here are the excerpt:

    npm ci will be significantly faster when:
    - There is a package-lock.json or npm-shrinkwrap.json file.

    In short, the main differences between using npm install and npm ci are:
    - The project must have an existing package-lock.json or npm-shrinkwrap.json.

**Comment by ljharb** [comment]
When the file is present, ci is faster than install. When the file is absent, install is your only option. What is the ambiguity?

**Comment by orjandh** [comment]
You are right! It did require a few reads before I got it, but it's probably just me.
